### PR TITLE
sig-docs meets weekly now

### PR DESCRIPTION
**Pull request by zoidbergwill**
In the minutes from the meetings: https://docs.google.com/document/d/1Ds87eRiNZeXwRBEbFr6Z7ukjbTow5RQcNZLaSvWWQsE/edit#heading=h.k7ir3bpr9sl6

### CODE REVIEW

File: sig-docs/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ To understand how this file is generated, see generator/README.md.
 Covers documentation, doc processes, and doc publishing for Kubernetes.
 
 ## Meetings
-* [Tuesdays at 17:30 UTC](https://zoom.us/j/678394311) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:30&tz=UTC).
+* [Tuesdays at 17:30 UTC](https://zoom.us/j/678394311) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=17:30&tz=UTC).
```
Comment (xiangpengzhao) [inline sig-docs/README.md]: Also modify the same thing in https://github.com/kubernetes/community/blob/master/sig-list.md please.

Comment (zoidbergwill) [inline sig-docs/README.md]: Thanks. Done!

**Comment by cblecker** [comment]
The authoritative place to change this is in `sigs.yaml` (https://github.com/kubernetes/community/blob/master/sigs.yaml#L291) and then run a "make".

@devin-donnelly @jaredbhatti Can you confirm that sig-docs now meets weekly? I skimmed the mailing list and last mention of frequency was back in January at biweekly.

**Comment by xiangpengzhao** [comment]
I remember in sig-docs meeting agenda there is something like "7.18 is the first weekly meeting". Please @devin-donnelly @jaredbhatti confirm this.

**Comment by zoidbergwill** [comment]
Awesome, thanks @cblecker. Did it in the right place now.

**Comment by xiangpengzhao** [comment]
@zoidbergwill please rebase your PR.

@cblecker Although the team leads didn't respond, we can still confirm this from the recent meetings of SIG docs: https://docs.google.com/document/d/1Ds87eRiNZeXwRBEbFr6Z7ukjbTow5RQcNZLaSvWWQsE/edit#

So I think it's okay to merge this PR after @zoidbergwill rebases.

**Comment by zoidbergwill** [comment]
I ran `make generate` and it also included `sig-ui` changes. :/

**Comment by xiangpengzhao** [comment]
@zoidbergwill I think it's because this PR https://github.com/kubernetes/community/pull/931 only modified  `sig-ui/README.md` but didn't modify `sigs.yaml`. I think you can modify `sigs.yaml` in your this PR according to the modification in #931 to help fix that.

cc @bryk

**Comment by zoidbergwill** [comment]
That looks right now.

Want me to rebase again with the changes into commits that make sense in master or not a big deal?

**Comment by xiangpengzhao** [comment]
I think you don't need to rebase if the bot doesn't complain for rebasing. But you'd better edit the PR title.

@bryk @cblecker WDYT?

**Comment by cblecker** [comment]
Going to get https://github.com/kubernetes/community/pull/937 in, and then you can rebase and remove the sig-ui updates

File: sig-cluster-lifecycle/README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Meeting recordings can be found [here](https://www.youtube.com/watch?v=ljK5dgSA7
 * [Slack](https://kubernetes.slack.com/messages/sig-cluster-lifecycle)
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
 
-## GitHub Teams
+## GitHub Teams
```
Comment (xiangpengzhao) [inline sig-cluster-lifecycle/README.md]: When I click the view button, here it shows as `## GitHub Teams`. It shows okay as a sub-title on the master branch. So I think you need to use what on the master branch here :)

File: sig-on-premise/README.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ Meeting recordings can be found [here](https://www.youtube.com/watch?v=dyUWqqNYU
 * [Slack](https://kubernetes.slack.com/messages/sig-onprem)
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-on-prem)
 
-## GitHub Teams
+## GitHub Teams
```
Comment (xiangpengzhao) [inline sig-on-premise/README.md]: and here.

File: sig-cluster-lifecycle/README.md (hunk 2 above)
Comment (zoidbergwill) [reply sig-cluster-lifecycle/README.md]: Not sure how those got updated, will change.

**Comment by xiangpengzhao** [comment]
@zoidbergwill please squash the commits :)

**Comment by zacharysarah** [comment]
Looks like I duplicated effort by accident: https://github.com/kubernetes/community/pull/1071/

I'd like to suggest closing this issue in favor of #1071 due to #1071's additional video link.

**Comment by cblecker** [comment]
Closing in favour of #1071

**Pull request closed without merging**